synapse/crypto/keyring.py

```diff
@@ -74,23 +74,19 @@ class VerifyJsonRequest:
         minimum_valid_until_ts: time at which we require the signing key to
             be valid. (0 implies we don't care)
 
-        request_name: The name of the request.
-
```

Comment:
hrm. this served a purpose: it was the thing that told us which event was failing verification.

Reply:
It was only used for some debug logging AFAICT? I can add that back in if you think its useful

Reply:
eh, it probably doesn't matter. It looks like we stick the event id back into the error message further up the stack.